source/org.jetbrains.kotlinx/kotlinx-serialization-protobuf-jvm/Additions/KotlinX.Serialization.Protobuf.ProtoNumberImpl.cs

```diff
@@ -0,0 +1,16 @@
+namespace KotlinX.Serialization.Protobuf 
+{
+
+	public sealed partial class ProtoNumberImpl
+    {
+		public global::Java.Lang.Class? AnnotationType ()
+		{
+			return default(global::Java.Lang.Class);
```

Comment:
Assuming we want to expose these `Impl` types -- which I'm still dubious of -- these method bodies are wrong, as they don't delegate to the Java side.